### ISSUE TITLE
DownloadDecisionMaker Couldn't process release.

### ISSUE THREAD

**Issue by ignoremenow** [Sonarr/Sonarr#1917]
Running version 2.0.0.4742 on linux and I receive this error on just one anime Attack On Titan.

Couldn't process release.: Scene Mappings contains a conflict for tvdbids 267440,267440. Please notify Sonarr developers.

NzbDrone.Core.DataAugmentation.Scene.InvalidSceneMappingException: Scene Mappings contains a conflict for tvdbids 267440,267440. Please notify Sonarr developers.
  at NzbDrone.Core.DataAugmentation.Scene.SceneMappingService.FindSceneMapping (System.String seriesTitle, System.String releaseTitle) [0x0001e] in M:\BuildAgent\work\b69c1fe19bfc2c38\src\NzbDrone.Core\DataAugmentation\Scene\SceneMappingService.cs:110 
  at NzbDrone.Core.DataAugmentation.Scene.SceneMappingService.FindTvdbId (System.String seriesTitle, System.String releaseTitle) [0x00000] in M:\BuildAgent\work\b69c1fe19bfc2c38\src\NzbDrone.Core\DataAugmentation\Scene\SceneMappingService.cs:76 
  at NzbDrone.Core.Parser.ParsingService.GetSeries (NzbDrone.Core.Parser.Model.ParsedEpisodeInfo parsedEpisodeInfo, System.Int32 tvdbId, System.Int32 tvRageId, NzbDrone.Core.IndexerSearch.Definitions.SearchCriteriaBase searchCriteria) [0x00002] in M:\BuildAgent\work\b69c1fe19bfc2c38\src\NzbDrone.Core\Parser\ParsingService.cs:264 
  at NzbDrone.Core.Parser.ParsingService.Map (NzbDrone.Core.Parser.Model.ParsedEpisodeInfo parsedEpisodeInfo, System.Int32 tvdbId, System.Int32 tvRageId, NzbDrone.Core.IndexerSearch.Definitions.SearchCriteriaBase searchCriteria) [0x0000d] in M:\BuildAgent\work\b69c1fe19bfc2c38\src\NzbDrone.Core\Parser\ParsingService.cs:131 
  at NzbDrone.Core.DecisionEngine.DownloadDecisionMaker+<GetDecisions>d__6.MoveNext () [0x0014a] in M:\BuildAgent\work\b69c1fe19bfc2c38\src\NzbDrone.Core\DecisionEngine\DownloadDecisionMaker.cs:78 
Close

**Comment by Taloth** [comment]
Tnx for reporting it, it's fixed. (I added the error so we'd be notified of stuff like this)

The series had an alias both on TheXEM and our own list, that's why you see the same tvdbid. I've fixed the specific duplicate, System->Tasks->Update Scene Mapping should refresh that immediately.
But the commit prevents similar error from appearing again if the tvdbid is the same, which we be included in the next develop build 10-20min from now.

@markus101 Pretty happy we haven't pushed out master yet :smile:
